astropy/coordinates/angle_formats.py

```diff
@@ -414,8 +414,11 @@ def degrees_to_dms(d):
 
 
 @deprecated(
-    "dms_to_degrees (or creating an Angle with a tuple) has ambiguous "
-    "behavior when the degree value is 0",
+    since="5.1",
+    message=(
+        "dms_to_degrees (or creating an Angle with a tuple) has ambiguous "
+        "behavior when the degree value is 0"
+    ),
     alternative=(
```

Comment:
`alternative` is not snown when this is used.

Reply:
Fixed. Thanks!